### PR TITLE
Querytimeout refactor - remove listeners

### PR DESCRIPTION
**Pull request by snimavat**

### CODE REVIEW

**Comment by basejump** [comment]
@snimavat this is a good idea. How do we set it for the users in controller?

**Comment by snimavat** [comment]
> @snimavat this is a good idea. How do we set it for the users in controller?

Made changes to EntityResponder

File: gorm-tools/src/main/groovy/gorm/tools/hibernate/GormToolsTrxManagerBeanPostProcessor.groovy [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-package gorm.tools.hibernate
-
-import groovy.transform.CompileStatic
-
-import org.grails.orm.hibernate.GrailsHibernateTransactionManager
-import org.springframework.beans.factory.annotation.Value
-import org.springframework.beans.factory.config.BeanPostProcessor
-
-/**
- * NOTE: THIS IS NOT USED BUT WE MIGHT COME BACK TO IT. KEPT FOR LATER USE
```
Comment (basejump) [inline gorm-tools/src/main/groovy/gorm/tools/hibernate/GormToolsTrxManagerBeanPostProcessor.groovy:14]: keep this for reference

**Review by basejump — CHANGES_REQUESTED** [review]
see https://github.com/yakworks/gorm-tools/pull/812 for start of refactor

**Pull request merged**